### PR TITLE
Upgrade Cert Manager

### PR DESCRIPTION
**Pull request by djsd123**
[Trello](https://trello.com/c/vK2e5D58)

Our current version of Cert Manager will be blocked by LetsEncrypt as of
1 Nov 2019.

Deploying Latest version of Cert Manager as of now (__31\10\2019__):
`0.11.0`

### CODE REVIEW

File: charts/cert-manager-resources/README.md [hunk 1]
```diff
@@ -7,37 +7,85 @@
 
 This chart requires that you first deploy the [Cert-Manager's](https://github.com/jetstack/cert-manager) [operators](https://coreos.com/blog/custom-resource-kubernetes-v17) prior to installing 
 
-To do so, install the [cert-manager](https://github.com/helm/charts/tree/master/stable/cert-manager) chart
+See official [chart](https://hub.helm.sh/charts/jetstack/cert-manager) for up to date deployment steps
```
Comment (xoen) [inline charts/cert-manager-resources/README.md]: did they move this out of the main `stable` repository?

File: charts/cert-manager-resources/README.md [hunk 2]
```diff
@@ -7,37 +7,85 @@
...
 
+```bash
+kubectl describe certificates cert-manager-resources --namespace default
 ```
-kubectl describe cert dev-prod-wildcard-cert --namespace=kube-system
+
+Once completed the resulting TLS certificate can be found in the default `namespace`
+
+```bash
+kubectl get secret cert-manager-resources-tls --namespace default
```
Comment (xoen) [inline charts/cert-manager-resources/README.md]: +1,000,000 for adding the commands to check if cert-manager is doing is job for the non-initiated @djsd123  ❤️

**Comment by xoen** [comment]
Only skimmed through it and didn't see anything out of the ordinary but wait for a Real Ops review CC: @sh31

File: charts/cert-manager-resources/README.md (hunk 1 above)
Comment (djsd123) [reply charts/cert-manager-resources/README.md]: Not sure if this chart still has a presence in the old stable repo. Most vendors do not use it anymore. When looking for a vendor chart, I always go to helm hub first.

**Comment by djsd123** [comment]
Cheers @obutterbach

**Pull request merged**